project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -2338,6 +2337,51 @@ Flux<RawDatasetInformationPxP> getRawDatasetInformationPxP(String projectId, Str
       int offset,
       int limit, SortRawData sorting, String filter);
 
+  /**
+   * Requests all available {@link RawDatasetInformationPxP} for a given experiment.
+   * <p>
+   * The request supports pagination.
+   *
+   * @param projectId    the identifier of the project the experiment belongs to
+   * @param experimentId the identifier of the experiment to query the measurements for
+   * @param offset       the offset value for the search to continue (pagination)
+   * @param limit        the maximum number of results to return (pagination)
+   * @param filter       a {@link RawDatasetFilter} for the request
+   * @return a reactive {@link Flux} of {@link RawDatasetInformationPxP}
+   * @since 1.12.0
+   */
+  Flux<RawDatasetInformationPxP> getRawDatasetInformationPxP(String projectId, String experimentId,
+      int offset,
+      int limit,
+      RawDatasetFilter filter);
+
+  /**
+   * Requests all available {@link RawDatasetInformationNgs} for a given experiment.
+   * <p>
+   * The request supports pagination.
+   *
+   * @param projectId    the identifier of the project the experiment belongs to
+   * @param experimentId the identifier of the experiment to query the measurements for
+   * @param offset       the offset value for the search to continue (pagination)
+   * @param limit        the maximum number of results to return (pagination)
+   * @param filter       a {@link RawDatasetFilter} for the request
+   * @return a reactive {@link Flux} of {@link RawDatasetInformationNgs}
+   * @since 1.12.0
+   */
+  Flux<RawDatasetInformationNgs> getRawDatasetInformationNgs(String projectId, String experimentId,
+      int offset,
+      int limit,
+      RawDatasetFilter filter);
+
+  record RawDatasetFilter(String filterTerm, List<SortOrder<RawDataSortingKey>> sortOrders) {
```

Comment:
If you want to make the filter extensible, you could thik of an appropriate interface for use in the methods. The record simply implements the interface. What do you think?